Content.Client/_NF/Atmos/Entities/GasRecycler/GasRecyclerArrow.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Client.Examine;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client._NF.Atmos.Entities.GasRecycler;
```

Comment:
```suggestion
namespace Content.Client._NF.Atmos.Systems;
```

Move this file to Content.Client/_NF/Atmos/Systems, and any client-side component to Content.Client/_NF/Atmos/Components.